.editorconfig

```diff
@@ -0,0 +1,18 @@
+# http://editorconfig.org/#file-format-details
+root = true
+
+# Always use Unix style new lines with new line ending on every file and trim whitespace
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# Python: PEP8 defines 4 spaces for indentation
+[*.py]
+indent_style = space
+indent_size = 4
+
+# Allow new paragrpah from two trailing whitespaces in markdown
```

Comment:
Actually, I don't even understand this comment. Is this the same as the trim_trailing_whitespace that is set below

Reply:
So for most files we want to remove trailing whitespace at the end of lines, as it is meaningless. However, in markdown files, having two spaces at the end of a line generates a new paragraph, so it is meaningful. Therefore we want to get rid of trailing whitespace in all files, apart from markdown files, as it might actually be useful there.

Reply:
Understood, thanks for the explanation